examples/gitlab-centos7-atomicapp/Nulecule

```diff
@@ -0,0 +1,63 @@
+---
+specversion: 0.0.2
+id: gitlab-atomicapp
+
+metadata:
+  name: Gitlab App
+  appversion: 1.2.0
+  description: Gitlab.
+graph:
+  - name: redis
```

Comment:
I agree with @goern here - showing people that they shouldn't reuse, but rather re-create is not best practice that we want to suggest.